bagit.xml

```diff
@@ -1,819 +1,800 @@
+              A sender-supplied identifier for the set, if any, of bags
+              to which it logically belongs.
+              This identifier must be unique across the sender's content, and if
+              recognizable as belonging to a globally unique scheme, the receiver
+              should make an effort to honor reference to it.
+          </t>
+          <t hangText="Bag-Count">
+              Two numbers separated by "of", in particular, "N of T",
+              where T is the total number of bags in a group of bags and N is the
+              ordinal number within the group; if T is not known, specify it as "?"
+              (question mark).  Examples:  1 of 2, 4 of 4, 3 of ?, 89 of 145.
+          </t>
+          <t hangText="Internal-Sender-Identifier">
+              An alternate sender-specific identifier for the content
+              and/or bag.
+          </t>
+          <t hangText="Internal-Sender-Description">
+              A sender-local prose description of the contents of the
+              bag.
+          </t>
+          </list>
+          </t>
+          <t>
+          In addition to these metadata elements, other arbitrary metadata
+          elements &may; also be present.
+          </t>
+          <figure>
+            <preamble>An example "bag-info.txt" file</preamble>
+            <artwork>
+Source-Organization: FOO University
+Organization-Address: 1 Main St., Cupertino, California, 11111
+Contact-Name: Jane Doe
+Contact-Phone: +1 111-111-1111
+Contact-Email: example@example.com
+External-Description: Uncompressed greyscale TIFF images from the
+      FOO papers colle...
+Bagging-Date: 2008-01-15
+External-Identifier: university_foo_001
+Payload-Oxum: 279164409832.1198
+Bag-Group-Identifier: university_foo
+Bag-Count: 1 of 15
+Internal-Sender-Identifier: /storage/images/foo
+Internal-Sender-Description: Uncompressed greyscale TIFFs created
+      from microfilm and are...
+          </artwork>
+          </figure>
+        </section>
+
+        <section title="Fetch File: fetch.txt" anchor="sec-fetch-file">
+
+          <t>
+            For reasons of efficiency, a bag &may; be sent with a list of files to be
+            fetched and added to the payload before it can meaningfully be checked
+            for completeness.  An &optional; tag file called the fetch file
+            contains such a list.
+          </t>
+
+          <t>
+            The fetch file &must; be named "fetch.txt". Every file listed in
+            the fetch file &must; be listed in every
+            payload manifest. A fetch file &must-not; list any tag files.
+          </t>
+          <t>
+            Each line of a fetch file &must; be of the form:
+          </t>
+          <figure>
+            <artwork>url length filename</artwork>
+            <postamble>
+              where <spanx style="emph">url</spanx> identifies the file to be
+              fetched and must be an absolute URI as defined in
```

Comment:
.. in addition RFC3986 requires ASCII `%` escapes of any unicode characters, while IRIs [RFC3987](https://www.ietf.org/rfc/rfc3987.txt) permits international characters, e.g. . We should be clear about which one we mean as we have UTF-8-mandated encoding also for fetch.txt.

E.g. which one we do require? 

I would prefer IRIs RFC3987:

```
http://résumé.example.org/café 1024 data/résumé/café.txt
```

vs the current URI requirement RFC3986:

```
http://xn--rsum-bpad.example.org/caf%C3%A9 1024 data/résumé/café.txt
```

(Note that IRI still permit `%`-style ASCII URIs):